src/main/kotlin/no/nav/klage/search/repositories/SaksbehandlerRepository.kt

```diff
@@ -61,12 +61,7 @@ class SaksbehandlerRepository(
     }
 
     private fun getYtelserForEnhet(enhet: Enhet): List<Ytelse> =
-        if (ytelserPerEnhet.containsKey(enhet.enhetId)) {
-            ytelserPerEnhet[enhet.enhetId]!!
-        } else {
-            logger.error("Fant ikke noen ytelse for enhet $enhet. Dette må legges til i kodebasen sporenstraks!")
-            emptyList()
-        }
+        klageenhetTilYtelser.filter { it.key.navn == enhet.enhetId }.flatMap { it.value }
```

Comment:
Mulig at det gir mening å bruke `find()` hvis man bare skal finne én.

Reply:
Enig, men jeg må ta det i en egen PR, for miljøet mitt er litt på bærtur nå